_sections/30-bindings.md

```diff
@@ -191,7 +191,7 @@ For convenience, the functions are categorized based on their main usage. Some f
 <a id="fn:indexed-repeat" href="#fn:indexed-repeat">`indexed-repeat(node-set arg, node-set repeat1, number index1, [node-set repeatN, number indexN]{0,2})`</a> | string | Returns a single node value from a node-set by selecting the 1-based index of a repeat node-set that this node is a child of. It does this up to 3 repeat levels deep. E.g. `indexed-repeat(//node, /path/to/repeat, //index1, /path/to/repeat/nested-repeat, //index2)` is meant to be a shortcut for `//repeat[position()=//index1]/nested-repeat[position()=index2]/node` in native XPath syntax.
 <a id="fn:Geographic-Functions" href="#fn:Geographic-Functions">**Geographic Functions**</a>|||
 <a id="fn:area" href="#fn:area">`area(node-set ns|geoshape gs)`</a> | number | Returns the calculated area in m2 of either a node-set of geopoints or a geoshape value (not a combination of both) on Earth. It takes into account the circumference of the Earth around the Equator but does not take altitude into account.
-<a id="fn:distance" href="#fn:distance">`distance(node-set ns|geoshape gs|geotrace gt)`</a> |  number | Returns the distance in meters of either a node-set of geopoints or a single geoshape value or a single geotrace value (not a combination of these) on Earth, in the sequence provided by the points in the parameter. It takes into account the circumference of the Earth around the Equator and does not take altitude into account.
+<a id="fn:distance" href="#fn:distance">`distance(node-set ns|geoshape gs|geotrace gt|(geopoint|string) arg*)`</a> |  number | Returns the distance in meters of either a single node-set of geopoints or a single geoshape value or a single geotrace value or a sequence of geopoint values (not a combination of these) on Earth. It takes into account the circumference of the Earth around the Equator and does not take altitude into account. The distance is computed in the order that the points are provided. The multiple-argument variant can take a combination of literal string geopoint values and references to geopoint nodes.
```

Comment:
It seems like everyone else already understands this except for me, but what does "distance" mean here? I'm guessing it's the total of the distance between subsequent points in the "sequence" being passed. If so, that was not obvious to me from this description? My instinct would be that a "distance" function only ever takes two args and then calculate the distance between them.

Reply:
> My instinct would be that a "distance" function only ever takes two args and then calculate the distance between them.

Yes, I agree that would be the typical assumption and easier for most people to understand which is why I think this addition is a good one.

I've added clarification on the algorithm, hopefully that is clearer now.

Reply:
Yeah that's great!